test/support/annotations/annotation_writer.rb

```diff
@@ -0,0 +1,36 @@
+module Annotations
+  class AnnotationWriter
+    def header_template
+      File.read(File.dirname(__FILE__) + '/templates/header.erb')
+    end
+
+    def annotation_template
+      File.read(File.dirname(__FILE__) + '/templates/annotation.erb')
+    end
+
+    def initialize(filename)
+      @file = File.open(filename, 'w')
+      @file.write(header_template)
+    end
+
+    def close
+      @file.close
+    end
+
+    def write(name, matched_annotations)
+      count = 1
+      output = "# #{name}\n"
+      matched_annotations.each do |annotation|
+        output += ERB.new(annotation_template).result(binding)
+        count += 1
```

Comment:
It looks like this count is arbitrarily being incremented. Where is it being used?